src/platform/user/profile/vet360/tests/components/VAPEditModalActionButtons.unit.spec.jsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
```

Comment:
This test file is largely identical to `src/platform/user/profile/vet360/tests/components/Vet360EditModalActionButtons.unit.spec.jsx` with minor edits as we're no longer rendering a button with a trash icon.